libraries/botbuilder-dialogs/src/dialogManager.ts

```diff
@@ -78,7 +78,7 @@ export class DialogManager extends Configurable {
         if (rootDialog) {
             this.rootDialog = rootDialog;
         }
-        this._dialogStateProperty = dialogStateProperty || 'DialogStateProperty';
+        this._dialogStateProperty = dialogStateProperty || 'DialogState';
```

Comment:
Big thanks to @joshgummersall for debugging this one, that prevented `MemoryAssertions` from properly inspecting the dialog state. [This matches .NET.](https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder.Dialogs/DialogManager.cs#L34)

Reply:
Should be `??` - glad to help as well!